src/lib/components/Treebank/config/config.js

```diff
@@ -1,110 +1,57 @@
-import smyth3 from './arethusa-configs/smyth3.json';
-import snap from './arethusa-configs/snap.json';
-import testlat from './arethusa-configs/testlat.json';
-import turkish from './arethusa-configs/turkish.json';
-import ud from './arethusa-configs/ud.json';
-import vortex from './arethusa-configs/vortex.json';
-import wheelock from './arethusa-configs/wheelock.json';
-import wheelock13 from './arethusa-configs/wheelock1-3.json';
-import wheelock15 from './arethusa-configs/wheelock15.json';
-import wheelock17 from './arethusa-configs/wheelock17.json';
-import wheelock22 from './arethusa-configs/wheelock22.json';
-import wheelock24 from './arethusa-configs/wheelock24.json';
-import wheelock30 from './arethusa-configs/wheelock30.json';
-import wheelock46 from './arethusa-configs/wheelock4-6.json';
-import whitaker from './arethusa-configs/whitaker.json';
-
 const configMap = {
-  'aldt-misc-grc': aldtMiscGrc,
-  'aldt-misc': aldtMisc,
-  'aldt2grc-stage': aldt2grcStage,
-  aldt2grc,
-  'aldt2lat-stage': aldt2latStage,
-  aldt2lat,
-  elmlatch1,
-  elmlatch10,
-  elmlatch10b,
-  elmlatch11,
-  elmlatch2,
-  elmlatch3,
-  elmlatch4,
-  elmlatch5,
-  elmlatch6,
-  elmlatch7,
-  elmlatch9,
-  english,
-  gclat,
-  harrington,
-  jmhgreek,
-  lyonGrec,
-  lyonLatin,
-  midlat,
-  morphgrc,
-  morphlat,
-  pedalion,
-  persian,
-  sanskrit,
-  smyth2,
-  smyth3,
-  snap,
-  testlat,
-  turkish,
-  ud,
-  vortex,
-  wheelock,
-  'wheelock1-3': wheelock13,
-  wheelock15,
-  wheelock17,
-  wheelock22,
-  wheelock24,
-  wheelock30,
-  'wheelock4-6': wheelock46,
-  whitaker,
+  'aldt-misc-grc': true,
+  'aldt-misc': true,
+  'aldt2grc-stage': true,
+  aldt2grc: true,
+  'aldt2lat-stage': true,
+  aldt2lat: true,
+  elmlatch1: true,
+  elmlatch10: true,
+  elmlatch10b: true,
+  elmlatch11: true,
+  elmlatch2: true,
+  elmlatch3: true,
+  elmlatch4: true,
+  elmlatch5: true,
+  elmlatch6: true,
+  elmlatch7: true,
+  elmlatch9: true,
+  english: true,
+  gclat: true,
+  harrington: true,
+  jmhgreek: true,
+  lyonGrec: true,
+  lyonLatin: true,
+  midlat: true,
+  morphgrc: true,
+  morphlat: true,
+  pedalion: true,
+  persian: true,
+  sanskrit: true,
+  smyth2: true,
+  smyth3: true,
+  snap: true,
+  testlat: true,
+  turkish: true,
+  ud: true,
+  vortex: true,
+  wheelock: true,
+  'wheelock1-3': true,
+  wheelock15: true,
+  wheelock17: true,
+  wheelock22: true,
+  wheelock24: true,
+  wheelock30: true,
+  'wheelock4-6': true,
+  whitaker: true,
 };
 
-const getConfig = (format, lang) => {
+const getConfig = (format, lang, callback) => {
   // Backwards compatibility
   const key = format === 'aldt' ? `aldt2${lang}` : format;
+  const config = configMap[key] ? key : (lang === 'grc' ? 'aldt-misc-grc' : 'aldt-misc')
 
-  if (configMap[key]) {
-    return configMap[key];
-  }
-
-  if (lang === 'grc') {
-    return aldtMiscGrc;
-  }
-
-  return aldtMisc;
+  import(`./arethusa-configs/${config}.json`).then(callback);
```

Comment:
This is the heart of the PR. Instead of having a bunch of imports at the top of the file, use the `import()` function and pass in a callback. When Webpack sees this, it creates separate bundle files for each of the JavaScript files not in the main import tree.